examples/network-hhmodel/main.rs

```diff
@@ -23,8 +26,13 @@ fn main() {
 
     // Load network
     network::init(&mut context, &people);
+    incidence_report::init(&mut context).unwrap();
 
-    let to_infect = vec![context.sample_person(MainRng, ()).unwrap()];
+    let to_infect: Vec<PersonId> = vec![context
+        .sample_person(MainRng, (AgeGroup, AgeGroupValue::Age18to64))
+        .unwrap()];
+    context.set_person_property(to_infect[0], InfectedBy, Some(to_infect[0]));
+    #[allow(clippy::vec_init_then_push)]
     seir::init(&mut context, &to_infect);
 
```

Comment:
This did not get pulled from the changes in `main.rs` right during the rebase.

```suggestion
    seir::init(&mut context, &to_infect);
```